reo/validators.py

```diff
@@ -440,7 +485,7 @@ def __init__(self, input_dict, ghpghx_inputs_validation_errors=None):
         self.input_as_none = []
         self.invalid_inputs = []
         self.resampled_inputs = []
-        self.emission_warning = []
+        self.emission_warning = [] ## I defined this the same way for NOx, might cause an error
```

Comment:
@adfarth same here

Reply:
Yes, this comment is outdated. Removed in [1efd149](https://github.com/NREL/REopt_Lite_API/pull/277/commits/1efd149dafd062deb009b460ccd0984a70ea9e4e)